nautobot/core/templates/generic/object_bulk_update.html

```diff
@@ -29,7 +29,9 @@ <h1>{% block title %}Editing {{ table.rows|length }} {{ obj_type_plural|betterti
     <div class="row">
         <div class="col-md-8">
             <div class="panel panel-default">
-                {% render_table table 'inc/table.html' %}
+                <div class="table-responsive">
+                    {% render_table table 'inc/table.html' %}
+                </div>
```

Comment:
Layout fix here - without it, in object bulk edit forms where the table of objects being edited was very wide, it would overflow its bounds instead of scrolling horizontally.